README.md

```diff
@@ -117,7 +117,7 @@ is used in the [Community managed DockerHub image](https://hub.docker.com/p/apac
 
 Visit the official Airflow website documentation (latest **stable** release) for help with
 [installing Airflow](https://airflow.apache.org/docs/apache-airflow/stable/installation.html),
-[getting started](https://airflow.apache.org/docs/apache-airflow/stable/start/index.html), or walking
+[getting started](https://airflow.apache.org/docs/apache-airflow/stable/start.html), or walking
```

Comment:
This will become a broken link until a version is released with these doc changes.

Mostly a note for committers

Reply:
We can add sphinx redirections.

Reply:
see `redirects.txt`

Reply:
Ah I see  it is added :)

Reply:
Fortunately the current docs already have a redirect from `start.rst` to `start/index.rst` - this PR contains a redirect that reverses that.

I think / hope this will result in a smooth transition 👍